model/src/main/kotlin/utils/SimplePackageConfigurationProvider.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model.utils
+
+import com.here.ort.model.Identifier
+import com.here.ort.model.Provenance
+import com.here.ort.model.config.PackageConfiguration
+import com.here.ort.model.readValue
+import java.io.File
+import java.io.FileFilter
+import java.util.LinkedList
+
+class SimplePackageConfigurationProvider(
+    configurations: List<PackageConfiguration> = emptyList()
+) : PackageConfigurationProvider {
+    companion object {
+        fun forDirectory(directory: File): SimplePackageConfigurationProvider {
+            val entries = directory.getChildrenRec().map { it.readValue<PackageConfiguration>() }
```

Comment:
I think you can use something like this instead of your custom function:

```kotlin
val entires = directory.walkBottomUp().filter { !it.isHidden }.map { it.readValue<PackageConfiguration>() }
```

Reply:
Done.